programs/bpf_loader/src/syscalls/cpi.rs

```diff
@@ -19,6 +14,25 @@ use {
     solana_transaction_context::BorrowedAccount,
     std::{mem, ptr},
 };
+// consts inlined to avoid solana-program dep
+const MAX_CPI_INSTRUCTION_DATA_LEN: u64 = 10 * 1024;
+#[cfg(test)]
+static_assertions::const_assert_eq!(
+    MAX_CPI_INSTRUCTION_DATA_LEN,
+    solana_program::syscalls::MAX_CPI_INSTRUCTION_DATA_LEN
+);
+const MAX_CPI_INSTRUCTION_ACCOUNTS: u8 = u8::MAX;
+#[cfg(test)]
+static_assertions::const_assert_eq!(
+    MAX_CPI_INSTRUCTION_ACCOUNTS,
+    solana_program::syscalls::MAX_CPI_INSTRUCTION_ACCOUNTS
+);
+const MAX_CPI_ACCOUNT_INFOS: usize = 128;
+#[cfg(test)]
+static_assertions::const_assert_eq!(
+    MAX_CPI_ACCOUNT_INFOS,
+    solana_program::syscalls::MAX_CPI_ACCOUNT_INFOS
+);
```

Comment:
Since we have an assertion here during tests, I believe the change must be fine. @Lichtso do you have any opinion on this?